.erb-lint.yml

```diff
@@ -0,0 +1,32 @@
+---
+exclude:
+  - '**/vendor/**/*'
```

Comment:
Without this exclusion, the erblint build in CI fails due to the existence of Ruby files in `vendor/bundle/`.